### PR TITLE
Update documentation for Grafter

### PR DESCRIPTION
**Pull request by cedric-cordenier**

### CODE REVIEW

File: source/includes/_policies.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Policies
+
+## /v2/policy/
+
+Note: this endpoint is currently in beta. The contract should not be considered stable, and is subject to change without notice.
```
Comment (danielinniss) [inline source/includes/_policies.md]: Can we not use _contract_ to indicate the data, "The contract should not be considered stable" is not a friendly or simple way of explaining the fact that the POST data expected and the response data may change.

File: source/includes/_policies.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+Creates a one-day fixed term Public Liability for the subject, the premium of which will be paid for by the work
```
Comment (danielinniss) [inline source/includes/_policies.md]:
```suggestion
Creates a one-day fixed term Public Liability policy for the subject, the premium of which will be paid for by the work
```

File: source/includes/_policies.md (hunk 1 above)
Comment (cedric-cordenier) [reply source/includes/_policies.md]: Done

File: source/includes/_policies.md (hunk 2 above)
Comment (cedric-cordenier) [reply source/includes/_policies.md]: Also done

**Review by danielinniss — APPROVED** [review]

**Pull request merged**